types/openpay/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
Why was `esModuleInterop` required?

Reply:
The Typescript Docs say the `esModuleInterop` flag is required to import the package as `import Openpay from 'openpay'`. But I think it can also be imported using `import Openpay = require('openpay')`, right? Then I think the flag is not required. I was a little confused about this...

Reply:
Yeah, it's super confusing.  Basically, you only want to use `esModuleInterop` in very specialized cases where the library works both with CommonJS-style imports (`const OpenPay = require`) and ES6-style imports (`import OpenPay from`), e.g. because it does fancy things with a bundler.  `import OpenPay = require` is a special TS-style import that "just works".

TL;DR: Please remove the flag if you're not aware of special characteristics of the library that make it necessary.

Reply:
Thank you for your clarification! Should I mark this conversation as resolved?